pallets/ethereum-events/src/lib.rs

```diff
@@ -106,8 +106,8 @@ mod test_parse_event;
 mod test_challenges;
 
 #[cfg(test)]
-#[path = "tests/test_set_ethereum_contract.rs"]
-mod test_set_ethereum_contract;
+#[path = "tests/test_map_nft_contract.rs"]
+mod test_map_nft_contract;
```

Comment:
This file name was changed from 'test_set_ethereum_contract' to 'test_map_nft_contract' which only performs tests for nft contract since the only remaining contract functionality within this pallet is for the nft contract.